src/poprox_recommender/data/eval.py

```diff
@@ -12,16 +12,21 @@
 class EvalData:
     name: str
 
-    def profile_truth(self, newsletter_id: UUID) -> pd.DataFrame | None: ...
+    def slate_truth(self, recommendation_id: UUID) -> pd.DataFrame | None: ...
 
-    def iter_profiles(self, *, limit: int | None = None) -> Generator[RecommendationRequest]: ...
-
-    def iter_profile_ids(self, *, limit: int | None = None) -> Generator[int | UUID]: ...
+    def iter_slate_ids(self, *, limit: int | None = None) -> Generator[int | UUID]: ...
 
     def lookup_request(self, id: int | UUID) -> RecommendationRequest: ...
```

Comment:
I think it will be clearer if we name the two lookup methods consistently: either both are `slate_truth` and `slate_request`, or `lookup_truth` and `lookup_request`.